examples/playlist/raw.go

```diff
@@ -74,12 +75,13 @@ func (k *rawPlaylistKind) GetVersions() []kindsys.VersionInfo {
 }
 
 // NOTE: this files are not used to do validation, but can be used generically to describe the kind
-func (k *rawPlaylistKind) GetJSONSchema(version string) (string, error) {
+func (k *rawPlaylistKind) GetOpenAPIDefinition(version string, ref common.ReferenceCallback) (common.OpenAPIDefinition, error) {
```

Comment:
I'm not super convinced about using a "k8s-specific" and "vendor" dep for part of a generic Kindsys interface (unaware of k8s), but as it looks like `common.OpenAPIDefinition` has no explicit references to k8s or any implementation detail other than OAPI/JSchema, I guess it could be fine enough